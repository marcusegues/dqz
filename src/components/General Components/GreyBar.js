// @flow
import React from 'react';
// $FlowFixMe
import { View } from 'react-native';
import { GREY_BAR } from '../../styles/colors';

export const GreyBar = () => (
  <View
    style={{
      width: 8,
      height: 4,
      alignSelf: 'flex-end',
      right: 24,
      backgroundColor: GREY_BAR,
    }}
  />
);
