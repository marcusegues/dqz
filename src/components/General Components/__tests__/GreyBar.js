import React from 'react';
import renderer from 'react-test-renderer';
import { GreyBar } from '../GreyBar';

describe('GreyBar', () => {
  test('renders according to snapshot', () => {
    const component = renderer.create(<GreyBar />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
