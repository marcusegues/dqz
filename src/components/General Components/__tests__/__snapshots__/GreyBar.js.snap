// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreyBar renders according to snapshot 1`] = `
<View
  style={
    Object {
      "alignSelf": "flex-end",
      "backgroundColor": "#818181",
      "height": 4,
      "right": 24,
      "width": 8,
    }
  }
/>
`;
